.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,21 +4,34 @@ about: Create a report to help us improve
 labels: bug
 ---
 
+<!--
+Checklist before submitting
+
+- [ ] Search through existing issue reports to check whether the issue already exists
+- [ ] Ensure you are running the latest software release, see https://github.com/linz/gazetteer/releases
+- [ ] Check QGIS/plugin log messages for more information.
+- [ ] If you can please create a light and self-contained sample dataset to reproduce the issue.
+-->
```

Comment:
It appears commented out sections do not render. Perhaps just provide the checklist?

Reply:
ignore that. I see what you are doing here. It is a good idea.